resources/sdk/pureclouddotnet/templates/ApiClient.mustache

```diff
@@ -275,6 +275,29 @@ namespace {{packageName}}.Client
                 return Convert.ToString (obj);
         }
 
+         /// <summary>
+        /// Creates a restclient with a base path string input
+        /// <returns>Returns a rest client</returns>
+        public RestClient withBasePath(String basePath){
```

Comment:
Name this method `setBasePath`. The "with" naming convention is used for builders, which this is not.